src/main/java/io/github/sefiraat/slimetinker/items/tinkermaterials/setup/Traits.java

```diff
@@ -0,0 +1,2620 @@
+
+    public static final MaterialTrait CORE_BRASS_ROD = new MaterialTrait()
+        .setPartType(MaterialTrait.PROP_ROD)
+        .setAddedBy(SupportedPluginsManager.CORE_NOTE)
+        .setTraitName("Oxygenated")
+        .setLore(
+            "Gives water breathing while held."
+        )
+        .addConsumer(TraitEventType.TICK, TickEvents::rodBrass);
+
+    public static final MaterialTrait CORE_BRASS_PLATES = new MaterialTrait()
+        .setPartType(MaterialTrait.PROP_PLATES)
+        .setAddedBy(SupportedPluginsManager.CORE_NOTE)
+        .setTraitName("Band")
+        .setLore(
+            "Durability loss + 200%. Damage reduced between 1-25%",
+            "getting lower as durability drops."
+        )
+        .addConsumer(TraitEventType.PLAYER_DAMAGED, PlayerDamagedEvents::plateBrass)
+        .addConsumer(TraitEventType.DURABILITY, DurabilityEvents::plateBrass);
+
+    public static final MaterialTrait CORE_BRASS_LINKS = new MaterialTrait()
+        .setPartType(MaterialTrait.PROP_LINKS)
+        .setAddedBy(SupportedPluginsManager.CORE_NOTE)
+        .setTraitName("Scarpaaarr!")
+        .setLore(
+            "Durability loss doubled. When damaged below 1/2 health, gain Speed 3."
+        )
+        .addConsumer(TraitEventType.PLAYER_DAMAGED, PlayerDamagedEvents::linksBrass);
+
+    public static final MaterialTrait CORE_ALUMINUM_BRASS_HEAD = new MaterialTrait()
+        .setPartType(MaterialTrait.PROP_HEAD)
+        .setAddedBy(SupportedPluginsManager.CORE_NOTE)
+        .setTraitName("Abra")
+        .setLore(
+            "Any hit mob has a % chance to be",
+            "randomly teleported."
+        )
+        .addConsumer(TraitEventType.ENTITY_DAMAGED, EntityDamageEvents::headAluBrass);
+
+    public static final MaterialTrait CORE_ALUMINUM_BRASS_ROD = new MaterialTrait()
+        .setPartType(MaterialTrait.PROP_ROD)
+        .setAddedBy(SupportedPluginsManager.CORE_NOTE)
+        .setTraitName("Studious")
+        .setLore(
+            "Tool and Player exp +50% but damage halved",
+            "and you are slower to act."
+        )
+        .addConsumer(TraitEventType.ENTITY_DAMAGED, EntityDamageEvents::rodAlubrass)
+        .addConsumer(TraitEventType.BLOCK_BREAK, BlockBreakEvents::rodAluBrass);
+
+    public static final MaterialTrait CORE_ALUMINUM_BRASS_PLATES = new MaterialTrait()
+        .setPartType(MaterialTrait.PROP_PLATES)
+        .setAddedBy(SupportedPluginsManager.CORE_NOTE)
+        .setTraitName("Magus")
+        .setLore(
+            "Magic Damage -50%"
+        )
+        .addConsumer(TraitEventType.PLAYER_DAMAGED, PlayerDamagedEvents::plateAluBrass);
+
+    public static final MaterialTrait CORE_ALUMINUM_BRASS_LINKS = new MaterialTrait()
+        .setPartType(MaterialTrait.PROP_LINKS)
+        .setAddedBy(SupportedPluginsManager.CORE_NOTE)
+        .setTraitName("Escape")
+        .setLore(
+            "Randomly teleport then hit (10s CD)"
```

Comment:
Consider having a consistent cd format. 
```suggestion
            "Randomly teleport then hit. (CD: 10s)"
```